### PR TITLE
Don't allow tests to pull docker images during testing

### PR DESCRIPTION
**Pull request by stefwalter**
We disable access to the various docker registries by updating /etc/hosts. Having the tests pull images just means more unreliability during testing.

### CODE REVIEW

**Comment by stefwalter** [comment]
Blocked on #4717

**Comment by dperpeet** [comment]
install failing on debian-unstable, fedora-atomic, rhel-atomic

**Comment by stefwalter** [comment]
They ran out of space, i'll check on them, and probably restrict pulling of the RHEL stuff on RHEL systems.

**Comment by stefwalter** [comment]
Needs different tags downloaded.

**Comment by stefwalter** [comment]
Images are still uploading, will retrigger those as they finish uploading.

**Comment by stefwalter** [comment]
Retriggering container/kubernetes to see if it's a bug in this change or a flake.

**Comment by stefwalter** [comment]
Rebasing

**Comment by stefwalter** [comment]
Debian builds seem to be failing here:

```
dh_install --list-missing -Xusr/src/debug
dh_install: Cannot find (any matches for) "lib/systemd/system/cockpit.service" (tried in "." and "debian/tmp")
dh_install: cockpit-ws missing files: lib/systemd/system/cockpit.service
dh_install: Cannot find (any matches for) "lib/systemd/system/cockpit.socket" (tried in "." and "debian/tmp")
dh_install: cockpit-ws missing files: lib/systemd/system/cockpit.socket
```

**Comment by stefwalter** [comment]
Rebasing. Rebuilding. Talked with @dperpeet about this on IRC.

File: test/images/scripts/continuous-atomic.setup [hunk 1]
```diff
@@ -33,12 +33,14 @@ rpm-ostree rebase centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/con
 
 ostree checkout centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous /var/local-tree
 
-# docker images that we need for integration testing
-/var/lib/testvm/docker-images.setup
+systemctl start docker || journalctl -u docker
 
-# we need cockpit/ws as well
+# we cockpit/ws as well
```
Comment (dperpeet) [inline test/images/scripts/continuous-atomic.setup]: weird change, but I don't think this needs to block a merge

Comment (dperpeet) [inline test/images/scripts/continuous-atomic.setup]: reverting this line during merge

**Pull request closed without merging**